app/models/partners/profile.rb

```diff
@@ -108,5 +115,22 @@ class Profile < Base
     def has_no_social_media?
       website.blank? && twitter.blank? && facebook.blank? && instagram.blank?
     end
+
+    def client_share_total
+      tot = 0
+      served_areas.each do |served_area|
+        tot += served_area.client_share
+      end
+      tot
+    end
+
+    def client_share_is_0_or_100
+      value = client_share_total
+      check = (value == 0 || value == 100)
+      if !check
+        errors.add(:base, "Total client share must be 0 or 100")
```

Comment:
I'd add some comments here describing the business logic behind why this is true.

Reply:
In next push.